### PR TITLE
Do not report unresolved symbols as errors in the web demo

### PR DESCRIPTION
**Pull request by chengluyu**

### CODE REVIEW

File: shared/src/main/scala/mlscript/codegen/Polyfill.scala [hunk 1]
```diff
@@ -76,7 +74,21 @@ object Polyfill {
...
+                JSTryStmt(
+                  (arg.member("constructor").member("name") +
+                    JSExpr(" ") +
+                    JSIdent("JSON").member("stringify")(arg, JSIdent("undefined"), JSIdent("2"))).`return` :: Nil,
+                  JSCatchClause(
+                    JSIdent("_"),
+                    JSIdent("String")(arg).`return` :: Nil,
+                  )
+                ) :: Nil,
+              ) :: Nil)
```
Comment (chengluyu) [inline shared/src/main/scala/mlscript/codegen/Polyfill.scala:91]: `JSON.stringify` reports errors for recursive objects. This is the plan B.

**Review by LPTK — CHANGES_REQUESTED** [review]
Thanks! I merged with the main branch. Please recompile the JS and push it to the PR.

**Comment by chengluyu** [comment]
Updated. Looks good.

**Review by LPTK — APPROVED** [review]
LGTM

**Pull request merged**